swiper.el

```diff
@@ -1240,11 +1240,11 @@ otherwise continue prompting for buffers."
       (t (buffer-file-name buffer)))))
 
 ;;* `swiper-all'
-(defun swiper-all-function (str)
-  "Search in all open buffers for STR."
+(defun swiper-all-buffers-function (str buffers)
+  "Search in all given BUFFERS for STR."
   (or
    (ivy-more-chars)
-   (let* ((buffers (cl-remove-if-not #'swiper-all-buffer-p (buffer-list)))
+   (let* ((buffers (cl-remove-if-not #'swiper-all-buffer-p buffers))
```

Comment:
This `let*` can be a plain `let`, right?

Reply:
yes, I don't see why not